src/Microsoft.Extensions.Diagnostics.HealthChecks/HealthReport.cs

```diff
@@ -14,10 +16,12 @@ public sealed class HealthReport
         /// Create a new <see cref="HealthReport"/> from the specified results.
         /// </summary>
         /// <param name="entries">A <see cref="IReadOnlyDictionary{TKey, T}"/> containing the results from each health check.</param>
-        public HealthReport(IReadOnlyDictionary<string, HealthReportEntry> entries)
+        /// <param name="totalDuration">A value indicating the total duration of executing <paramref name="entries"/>.</param>
```

Comment:
Suggestion (use the wording your used on the property):

> A value indicating the total duration of executing a group of <see cref="IHealthCheck"/> instances.

I think it's good that we define this as the time the health check service took to execute.